js/app/src/components/StatusTracker/Toast.svelte

```diff
@@ -1,17 +1,47 @@
 <script lang="ts">
 	import { flip } from "svelte/animate";
-	import Error from "./Error.svelte";
 	import type { ToastMessage } from "./types";
+	import ToastContent from "./ToastContent.svelte";
+	import type { ThemeMode } from "../types";
 
+	export let theme_mode: ThemeMode;
 	export let messages: ToastMessage[] = [];
 </script>
 
 <div class="toast-wrap">
 	{#each messages as { type, message, id } (id)}
 		<div animate:flip={{ duration: 300 }} style:width="100%">
-			<!-- {#if type === "error"} -->
-			<Error {message} on:close {id} />
-			<!-- {/if} -->
+			{#if type === "warning"}
+				<ToastContent
```

Comment:
let's just have one ToastContent with different theme_color and description instead of three different ToastContents